modular_citadel/code/game/gamemodes/gangs/gang_items.dm

```diff
@@ -0,0 +1,410 @@
+	return TRUE
+
+/datum/gang_item/clothing/suit
+	name = "Gang Armored Outerwear"
+	id = "suit"
+	cost = 1
+
+/datum/gang_item/clothing/suit/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	if(gang.outer_outfits.len)
+		var/outfit = pick(gang.outer_outfits)
+		if(outfit)
+			var/obj/item/O = new outfit(user.loc)
+			O.armor = O.armor.setRating(melee = 20, bullet = 35, laser = 10, energy = 10, bomb = 30, bio = 0, rad = 0, fire = 30, acid = 30)
+			O.desc += " Tailored for the [gang.name] Gang to offer the wearer moderate protection against ballistics and physical trauma."
+			user.put_in_hands(O)
+			to_chat(user, "<span class='notice'> This is your gang's official outerwear, wearing it will increase your influence")
+			return
+	return TRUE
+
+
+/datum/gang_item/clothing/hat
+	name = "Pimp Hat"
+	id = "hat"
+	cost = 16
+	item_path = /obj/item/clothing/head/collectable/petehat/gang
+
+
+/obj/item/clothing/head/collectable/petehat/gang
+	name = "pimpin' hat"
+	desc = "The undisputed king of style."
+
+/datum/gang_item/clothing/mask
+	name = "Golden Death Mask"
+	id = "mask"
+	cost = 18
+	item_path = /obj/item/clothing/mask/gskull
+
+/obj/item/clothing/mask/gskull
+	name = "golden death mask"
+	icon_state = "gskull"
+	desc = "Strike terror, and envy, into the hearts of your enemies."
+
+/datum/gang_item/clothing/shoes
+	name = "Bling Boots"
+	id = "boots"
+	cost = 22
+	item_path = /obj/item/clothing/shoes/gang
+
+/obj/item/clothing/shoes/gang
+	name = "blinged-out boots"
+	desc = "Stand aside peasants."
+	icon_state = "bling"
+
+/datum/gang_item/clothing/neck
+	name = "Gold Necklace"
+	id = "necklace"
+	cost = 9
+	item_path = /obj/item/clothing/neck/necklace/dope
+
+/datum/gang_item/clothing/hands
+	name = "Decorative Brass Knuckles"
+	id = "hand"
+	cost = 11
+	item_path = /obj/item/clothing/gloves/gang
+
+/obj/item/clothing/gloves/gang
+	name = "braggadocio's brass knuckles"
+	desc = "Purely decorative, don't find out the hard way."
+	icon_state = "knuckles"
+	w_class = 3
+
+/datum/gang_item/clothing/belt
+	name = "Badass Belt"
+	id = "belt"
+	cost = 13
+	item_path = /obj/item/storage/belt/military/gang
+
+/obj/item/storage/belt/military/gang
+	name = "badass belt"
+	icon_state = "gangbelt"
+	item_state = "gang"
+	desc = "The belt buckle simply reads 'BAMF'."
+
+///////////////////
+//WEAPONS
+///////////////////
+
+/datum/gang_item/weapon
+	category = "Purchase Weapons:"
+
+/datum/gang_item/weapon/ammo
+
+/datum/gang_item/weapon/shuriken
+	name = "Shuriken"
+	id = "shuriken"
+	cost = 2
+	item_path = /obj/item/throwing_star
+
+/datum/gang_item/weapon/switchblade
+	name = "Switchblade"
+	id = "switchblade"
+	cost = 5
+	item_path = /obj/item/switchblade
+
+/datum/gang_item/weapon/surplus
+	name = "Surplus Rifle"
+	id = "surplus"
+	cost = 6
+	item_path = /obj/item/gun/ballistic/automatic/surplus
+
+/datum/gang_item/weapon/ammo/surplus_ammo
+	name = "Surplus Rifle Ammo"
+	id = "surplus_ammo"
+	cost = 5
```

Comment:
Prob can be lowerd to 2 or 3 safely

Reply:
Lowered seeing as the surplus rifle has seen little use.